docs/data/date-pickers/lifecycle/lifecycle.md

```diff
@@ -137,6 +135,34 @@ Clicking on any built-in button of the action bar will close the picker.
 Clicking on a shortcut will close the picker, except if the `changeImportance` property has been set to `"set"` instead of the default value `"accept"`.
 You can find more information [in the dedicated doc section](/x/react-date-pickers/shortcuts/#behavior-when-selecting-a-shortcut).
 
+## Lifecycle on pickers: `onChange`
+
+### Usage
+
+The `onChange` callback is called whenever the current value changes.
+
+If you don't want to listen to the intermediary steps, consider using the [`onAccept` prop](/x/react-date-pickers/lifecycle/#onaccept) instead.
```

Comment:
It seems to me that the hyperlink here is redirecting to the lifecycle main page 🤔

Reply:
Good catch, I did not update after renaming the section